ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/transactions/sorter/AbstractPendingTransactionsSorter.java

```diff
@@ -65,10 +66,8 @@
 /**
  * Holds the current set of pending transactions with the ability to iterate them based on priority
  * for mining or look-up by hash.
- *
- * <p>This class is safe for use across multiple threads.
```

Comment:
Did something change to make this no longer thread safe?

Reply:
No, removed by error and readded

Reply:
checked it's readded.